rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -241,15 +239,16 @@ A node MUST make sure to only add transactions of which all previous transaction
 
 The `State` message is sent as response to various conditions of the gossip protocol (see §5).
 The `State` message MUST contain a new `conversationID`.
-The `State` message contains a `XOR` value and an `LC`.
+The `State` message contains an `XOR` value and an `LC`.
```

Comment:
To avoid confusion on what to send in response to a Gossip message, I think we should make clear that the XOR and LC reflect the current state of the local DAG, so not the temporary XOR calculated when parsing the Gossip msg.